### ISSUE TITLE
set_sequence_bit_depth claims bogus bit depth is set

### ISSUE THREAD

**Issue by hmaarrfk** [sci-microscopy/illuminate#5]
```python
print(i.set_sequence_bit_depth(16))
ERROR - invalid bit depth! (allowed values are 1 or 8)
New sequence bit depth is: 16
```

**Comment by hmaarrfk** [comment]
Maybe it is an experimental feature?

Seems like `parameters_json` also reports that the bit depth is 16:

```json
{
...
"color_channels" : ["r", "g", "b"],
"color_channel_center_wavelengths" : {"r" : 0.480, "g" : 0.525, "b" : 0.625},
"trigger_input_count" : 1,
"trigger_output_count" : 1,
"bit_depth" : 16,
...
}
```

**Comment by zackphillips** [comment]
Yeah this is experimental but fairly stable. I use it nearly every day for my Ph.D. work but it hasn't been tested for anything outside of my research problem. We could also add 16-bit sequences in the future but I haven't had the need for them yet, 8-bit and binary (1-bit) seem to work file for most  

Fixed in 331875b

**Comment by hmaarrfk** [comment]
I see, thanks for the explanation!

I'll have to try it out soon.
